test-bench-utils/public/views/xxe.ejs

```diff
@@ -17,17 +8,19 @@
 </p>
 <% sinkData.forEach(function(sink) { %>
 <div class="row">
-  <div class="col-sm-10 col-sm-8" style="padding-bottom: 30px;">
+  <div class="col-xs-12 col-sm-6" style="padding-bottom: 30px;">
     <h4 class="sub-header">
       <code><%= sink.name %></code>
     </h4>
-    <!-- this rule is input less putting the attack vector here just for context -->
-    <textarea name="input" class="form-control" id="attack-xml"><%= input %></textarea>
     <form method="<%= sink.method %>" action="<%= sink.url %>/unsafe" target="_blank">
-      <div class="form-group"></div>
+      <div class="form-group">
+        <label>SELECT "${input}" as "test";</label>
```

Comment:
what does this mean?

Reply:
Whoops - that was probably just copy pasta from another template.